src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,55 @@
+@import './variables';
+
+@mixin product-title-font {
+  font-family: $card-ff;
+  color: $text-color;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin product-code-font {
+  font-family: $card-ff;
+  color: $text-color;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
It seems there's a redundancy between the `product-code-font` and `product-code` mixins (lines 11-16 and 19-24). They both set font properties for product codes. Ensure that each mixin has a unique purpose and avoid duplication.